src/main/java/atwoz/atwoz/match/command/domain/match/Match.java

```diff
@@ -0,0 +1,47 @@
+package atwoz.atwoz.match.command.domain.match;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Entity
+@Table(name = "matches", indexes = {
+        @Index(name = "idx_requester_id", columnList = "requesterId"),
+        @Index(name = "idx_responder_id", columnList = "responderId")
+})
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Builder(access = AccessLevel.PRIVATE)
+public class Match {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Getter
+    private Long requesterId;
+
+    @Getter
+    private Long responderId;
+
+    @Getter
+    private String requestMessage;
+
+    @Getter
+    private String responseMessage;
+
+    @Enumerated(EnumType.STRING)
+    @Column(columnDefinition = "varchar(50)")
+    private MatchStatus status;
+
+    public static Match requestMatching(@NonNull Long requesterId, @NonNull Long responderId, @NonNull String requestMessage) {
```

Comment:
```suggestion
    public static Match request(long requesterId, long responderId, @NonNull String requestMessage) {
```

메서드 명이랑 파라미터 타입 바꾸면 가독성이 더 좋아질것 같아요

Reply:
네 @NonNull 대신 원시타입으로 사용할 수 있는 것은 해당 방식으로 변경하였습니다!